src/test/fuzz/rbf.cpp

```diff
@@ -149,7 +154,9 @@ FUZZ_TARGET(package_rbf, .init = initialize_package_rbf)
             mempool_txs.pop_back();
             break;
         }
-        AddToMempool(pool, child_entry);
+        if (!pool.GetIter(child_entry.GetTx().GetHash())) {
```

Comment:
belt-and-suspenders are fine, but it "shouldn't happen" since each tx chain is sourced from a new `g_outpoints`?

Reply:
No I think this one is just "belt" -- if you look at lines 146-148 above, we're not always modifying the "child" transaction here, so sometimes it'll be a duplicate of something we already have.

Reply:
ah right